components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -388,4 +406,38 @@ private Object getBlobObject(InputStream is) throws IdentityApplicationManagemen
         return null;
     }
 
+    public Timestamp getTimeStamp(String key, String type) {
+        boolean isExist = isExist(key, type);
+
+        if (isExist) {
+            Connection connection = null;
+            PreparedStatement preparedStatement = null;
+            ResultSet resultSet = null;
+            Timestamp timestamp = null;
+            try {
+                connection = jdbcPersistenceManager.getDBConnection();
+                connection.setAutoCommit(false);
+                preparedStatement = connection.prepareStatement(SQL_SELECT_TIME_CREATED);
+                preparedStatement.setString(1, key);
+                preparedStatement.setString(2, type);
+                resultSet = preparedStatement.executeQuery();
+                if (resultSet.next()) {
+                    timestamp = resultSet.getTimestamp(1);
```

Comment:
Make it a habit to use "Column names" instead of "indices" to make the code less error-prone as well as more readable.